packages/components/pie-card/src/index.ts

```diff
@@ -53,28 +53,36 @@ export class PieCard extends LitElement implements CardProps {
     @validPropertyValues(componentSelector, paddingValues, undefined)
     public padding?: CardProps['padding'];
 
+    private onClickHandler (event: Event) {
+        if (this.disabled) {
+            event.preventDefault();
+            event.stopPropagation();
+        }
+    }
+
     /**
      * Renders the card as an anchor element.
      *
      * @private
      */
     private renderAnchor (classes: ClassInfo): TemplateResult {
         const paddingCSS = this.generatePaddingCSS();
+        const {
+            href, rel, target, disabled, aria,
+        } = this;
 
         return html`
             <a
                 class="${classMap(classes)}"
                 data-test-id="pie-card"
-                ?disabled=${this.disabled}
-                href=${this.href || ''}
-                target=${this.target || nothing}
-                rel=${this.rel || nothing}
+                href=${ifDefined(href && !disabled ? href : undefined)}
```

Comment:
I think here you can just check if it not disabled i.e `!disabled ? href : undefined` because we only really care about the disabled state and not the href provided as it will be set after the `?` anyway.